src/main/scss/components/_dropdowns.scss

```diff
@@ -281,3 +281,11 @@ $dropdown-padding: 0.375rem;
     }
   }
 }
+
+.jenkins-breadcrumbs__overflow-item {
+  display: block;
+  overflow: hidden;
+  max-width: 25ch;
+  text-overflow: ellipsis;
```

Comment:
I'm not sure that we should be overflowing so early in the overflow menu, there's lots of space left and there's no way currently to get the full name that I can see:

<img width="347" height="229" alt="Image" src="https://github.com/user-attachments/assets/662a5dbf-2eeb-4688-8267-d43bac7de89b" />

Reply:
You get a tooltip if you hover it.

Reply:
I only get a tooltip on the first element not the second:

<img width="264" height="229" alt="Image" src="https://github.com/user-attachments/assets/8c8bc3d3-65be-43a3-9b1a-60f6d000397c" />

This is my folder chain:

<img width="981" height="63" alt="Image" src="https://github.com/user-attachments/assets/861d928f-1680-4d9b-9f4f-8326e69ce786" />

tooltip working:
<img width="326" height="165" alt="Image" src="https://github.com/user-attachments/assets/f386b4a7-9598-4ac4-a22b-ffe455c2ebba" />

Reply:
This doesn't work if you navigate between pages the tooltip only works after a page refresh 

Also appears to only work at a certain size of the screen, in the video below it didn't work after a page refresh but if I made it smaller it did with page refresh only and not when navigating:

<img width="256" height="87" alt="image" src="https://github.com/user-attachments/assets/cf9911b3-3290-4056-9bfb-9cdd78e79c66" />


https://github.com/user-attachments/assets/a6d067f9-415a-4f59-8139-ed0fede80ee8

Reply:
Logging inside of `registerTooltip` shows its not being called:

<img width="867" height="202" alt="image" src="https://github.com/user-attachments/assets/7dd0ea5f-53da-4a70-97cd-85f8b0827004" />

Reply:
Have a look and see what you think, I'm not sure whats going on here but we could leave this I guess.